### PR TITLE
Support new wording of V4a "local health" item

### PR DESCRIPTION
**Pull request by nmdefries**
In Wave 8, the second subitem of V4a was switched from asking about "Local health workers" to "Doctors and other health professionals you go to for medical care". This resulted in a sharp increase in reported trust that is misleading when the new and old versions are reported under the same signal name.

This change describes the new V4a "doctors" signal and discontinues the previous "local health" version.

### CODE REVIEW

File: docs/api/covidcast-signals/fb-survey.md [hunk 1]
```diff
@@ -393,10 +393,11 @@ Respondents who indicate that "I don't believe I need a COVID-19 vaccine" (in it
...
 | `smoothed_wvaccine_likely_govt_health` | Estimated percentage of respondents who would be more likely to get a COVID-19 vaccine if it were recommended to them by government health officials, among respondents who have not yet been vaccinated. <br/> **Earliest date available:** 2021-01-20 | V4 |
 | `smoothed_wvaccine_likely_politicians` | Estimated percentage of respondents who would be more likely to get a COVID-19 vaccine if it were recommended to them by politicians, among respondents who have not yet been vaccinated. <br/> **Earliest date available:** 2021-01-20 | V4 |
+| `smoothed_wvaccine_likely_doctors` | Estimated percentage of respondents who would be more likely to get a COVID-19 vaccine if it were recommended to them health professionals they go to for medical care, among respondents who have not yet been vaccinated. This item was shown to respondents starting in Wave 8. <br/> **Earliest date available:** 2021-02-08 | V4 |
```
Comment (capnrefsmmat) [inline docs/api/covidcast-signals/fb-survey.md:400]:
```suggestion
| `smoothed_wvaccine_likely_doctors` | Estimated percentage of respondents who would be more likely to get a COVID-19 vaccine if it were recommended to them by doctors and other health professionals they go to for medical care, among respondents who have not yet been vaccinated. This item was shown to respondents starting in Wave 8. <br/> **Earliest date available:** 2021-02-08 | V4 |
```

**Review by capnrefsmmat — CHANGES_REQUESTED** [review]
one wording fix, otherwise good

**Review by capnrefsmmat — APPROVED** [review]
looks good; let's merge when the reissue is done

**Comment by krivard** [comment]
I believe this should be good to go

**Pull request merged**